sonic-utilities-tests/show_ip_int_test.py

```diff
@@ -0,0 +1,156 @@
+
+show_multi_asic_ip_intf_all = """\
+Interface        Master    IPv4 address/mask    Admin/Oper    BGP Neighbor    Neighbor IP
+---------------  --------  -------------------  ------------  --------------  -------------
+Loopback0                  40.1.1.1/32          error/down    N/A             N/A
+Loopback4096               1.1.1.1/24           error/down    N/A             N/A
+                           2.1.1.1/24                         N/A             N/A
+PortChannel0001            20.1.1.1/24          error/down    T2-Peer         20.1.1.5
+PortChannel0002            30.1.1.1/24          error/down    T0-Peer         30.1.1.5
+veth@eth1                  192.1.1.1/24         error/down    N/A             N/A
+veth@eth2                  193.1.1.1/24         error/down    N/A             N/A"""
+
+show_multi_asic_ipv6_intf_all = """\
+Interface        Master    IPv4 address/mask                       Admin/Oper    BGP Neighbor    Neighbor IP
+---------------  --------  --------------------------------------  ------------  --------------  -------------
+Loopback0                  fe80::60a5:9dff:fef4:1696%Loopback0/64  error/down    N/A             N/A
+PortChannel0001            aa00::1/64                              error/down    N/A             N/A
+                           fe80::80fd:d1ff:fe5b:452f/64                          N/A             N/A
+PortChannel0002            bb00::1/64                              error/down    N/A             N/A
+                           fe80::80fd:abff:fe5b:452f/64                          N/A             N/A"""
+
+show_error_invalid_af = """Invalid argument -a ipv5"""
+
+
+@pytest.fixture(scope="class")
+def setup_teardown_single_asic():
+    os.environ["PATH"] += os.pathsep + scripts_path
+    os.environ["UTILITIES_UNIT_TESTING"] = "2"
+    os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = ""
+    yield
+
+    os.environ["UTILITIES_UNIT_TESTING"] = "0"
+
+
+@pytest.fixture(scope="class")
+def setup_teardown_multi_asic():
+    os.environ["PATH"] += os.pathsep + scripts_path
+    os.environ["UTILITIES_UNIT_TESTING"] = "2"
+    os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = "multi_asic"
+    yield
+    os.environ["UTILITIES_UNIT_TESTING"] = "0"
+    os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = ""
+
+
+def verify_output(output, expected_output):
+    lines = output.splitlines()
+    ignored_intfs = ['eth0', 'lo']
+    for intf in ignored_intfs:
+        # the output should have line to display the ip address of eth0 and lo
+        assert len([line for line in lines if line.startswith(intf)]) == 1
+
+    new_output = '\n'.join([line for line in lines if not any(i in line for i in ignored_intfs)])
+    print(new_output)
+    assert new_output == expected_output
+
+
+@pytest.mark.usefixtures('setup_teardown_single_asic')
+class TestShowIpInt(object):
+
+    def test_show_ip_intf_v4(self):
+        return_code, result = get_result_and_return_code(" ipintutil")
```

Comment:
Delete space between `"` and `ipintutil`

Reply:
Fixed in latest commit